### PR TITLE
Update hue.ini and simpler naming/messages

### PR DESCRIPTION
**Pull request by Harshg999**
## What changes were proposed in this pull request?

- Slack integration added in hue.ini and made naming/message of slack integration simpler and generic.

### CODE REVIEW

File: desktop/conf.dist/hue.ini [hunk 1]
```diff
@@ -814,6 +814,14 @@
    ## collection_interval=30000
 
 
+  # Configuration options for Slack
+  # ------------------------------------------------------------------------
+  [[slack]]
+
+   # Turns on Slack application API endpoints
+   ## is_enabled=True
```
Comment (romainr) [inline desktop/conf.dist/hue.ini:822]: nit: True --> true

(file convention)

File: desktop/core/src/desktop/conf.py [hunk 2]
```diff
@@ -707,9 +707,9 @@ def get_deprecated_login_lock_out_by_combination_browser_user_agent():
   key='slack',
   help=_("""Configuration options for slack """),
   members=dict(
-    ENABLE_SLACK_INTEGRATION=Config(
-      key='enable_slack_integration',
-      help=_('Enable slack bot server URLs "/slack/"'),
+    IS_ENABLED=Config(
+      key='is_enabled',
+      help=_('Turns on Slack application API endpoints'),
       default=True,
```
Comment (romainr) [inline desktop/core/src/desktop/conf.py:713]: True --> False

File: desktop/conf.dist/hue.ini (hunk 1 above)
Comment (romainr) [reply desktop/conf.dist/hue.ini:822]: and actually let's put 'false', as this should be the default for now

**Review by romainr — COMMENTED** [review]
Nice!

File: desktop/core/src/desktop/conf.py (hunk 2 above)
Comment (Harshg999) [reply desktop/core/src/desktop/conf.py:713]: Okay

**Pull request merged**